src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj

```diff
@@ -45,4 +55,20 @@
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Bcl.AsyncInterfaces\src\Microsoft.Bcl.AsyncInterfaces.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">
```

Comment:
Add this to the top property group instead:

```xml
    <!-- Use targeting pack references instead of granular ones in the project file. -->
    <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
```

Then you don't need to list each reference here.

We use this elsewhere. Ex:

https://github.com/dotnet/runtime/blob/6168c4afcba55ebf900c25c61841a32b11591498/src/libraries/Microsoft.Extensions.Hosting/src/Microsoft.Extensions.Hosting.csproj#L7-L8